backend/test/controllers/user.test.js

```diff
@@ -0,0 +1,53 @@
+const { getAll, getUser, addUser, updateUser, deleteUser } = require("../../src/controllers/user");
+
+describe("Testing user controllers", () => {
+    it("Gets an array of all users", async () => {
```

Comment:
i'm not sure about the naming of these, but i feel like the description was off somehow, and this is more explanitory
`    it("Checks to see if it is an array of basic user objects", async () => {`